djangocms_navigation/templates/admin/djangocms_navigation/main_navigation_confirmation.html

```diff
@@ -0,0 +1,42 @@
+{% extends "admin/base_site.html" %}
+{% load i18n admin_urls static %}
+{% block title %}{% trans "Revert Confirmation" %}{% endblock %}
+
+{% block extrahead %}
+    {{ block.super }}
+    {{ media }}
+    <script type="text/javascript" src="{% static 'admin/js/cancel.js' %}"></script>
+{% endblock %}
+
+{% block breadcrumbs %}{% endblock %}
+{% block bodyclass %}{{ block.super }} app-{{ opts.app_label }} model-{{ opts.model_name }} main-navigation-confirmation{% endblock %}
+
+{% block content %}
+<h2>{% trans "Change Main Navigation" %}</h2>
+<h3>{{ object_name }}</h3>
+    <h4>{% blocktrans %}Are you sure you would like to set the menu: <u>{{ menucontent }}</u> as the main navigation?{% endblocktrans %}</h4>
+<div>
+{% if extra_context.existing_menus %}
+    <h4>By doing this, the existing navigation below will no longer be the main navigation.</h4>
+<br>
+<ul>
+    {% for identifier in extra_context.existing_menus %}
+        <li>
+        <p>{{identifier}}</p>
```

Comment:
```suggestion
        <p>{{ identifier }}</p>
```
Sorry to be picky but for consistency I would add spacing